ENGAGEHF/Supporting Files/ENGAGEHF.entitlements

```diff
@@ -1,10 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict>
-	<key>com.apple.developer.applesignin</key>
-	<array>
-		<string>Default</string>
-	</array>
-</dict>
+<dict/>
```

Comment:
I think this might need to be re-added before the merge?

Reply:
Ups, good catch 👍